src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -255,6 +255,31 @@ export class SearchResults extends Component {
           maxPageListLength={perPage}
           showLastPage
         />
+
+        {/* Helpful Links */}
```

Comment:
I think I would expect the helpful links section to be a part of the `SearchResultsPage` component and not the `SearchResults` components as these are not actually search results. Since the wrapping element is just a fragment, I think we should be able to move it below line 64 in `src/applications/yellow-ribbon/components/SearchResultsPage/index.js`

Reply:
Ah yes, thanks for pointing that out!